### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by PhilipMathieu** [PhilipMathieu/unet-orthoimagery#8]

**Comment by PhilipMathieu** [comment]
Definitely:
1. Training batch size [16 -> Jack] [32 -> James] [64,128 -> Philip]
2. Learning rate [1e-1, 1e-2, 1e-3, 1e-4, 1e-5]
3. Epochs - start with 20 for each combination, then more for working subsets
4. Amount of data augmentation 4x

Single test with best model:
1. Non-augmented dataset
2. Without DEM (3-channel)
3. With IR (5-channel)

Future:
- Momentum
- Bilinear vs Cubic upscaling
- Learning rate scheduler/constant
- Other datasets

Assess performance by training loss + validation Dice coefficient
